<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="2" failed="0" skipped="2">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-01-31T23:25:31 PKT" name="Default suite" finished-at="2022-01-31T23:25:32 PKT" duration-ms="876">
    <groups>
    </groups>
    <test started-at="2022-01-31T23:25:31 PKT" name="Default test" finished-at="2022-01-31T23:25:32 PKT" duration-ms="876">
      <class name="endToEndAutomation.CommonDataProviderXl">
        <test-method signature="commonDataProvider(java.util.Hashtable)[pri:0, instance:endToEndAutomation.CommonDataProviderXl@1b2abca6]" started-at="2022-01-31T23:25:32 PKT" name="commonDataProvider" data-provider="dp" finished-at="2022-01-31T23:25:32 PKT" duration-ms="4" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{class1=test, name=ldfjlds, code=test3, runmode=Y}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- commonDataProvider -->
        <test-method signature="commonDataProvider(java.util.Hashtable)[pri:0, instance:endToEndAutomation.CommonDataProviderXl@1b2abca6]" started-at="2022-01-31T23:25:32 PKT" name="commonDataProvider" data-provider="dp" finished-at="2022-01-31T23:25:32 PKT" duration-ms="1" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{class1=test5, name=test4, code=test6, runmode=N}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[with N no]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: with N no
at endToEndAutomation.CommonDataProviderXl.commonDataProvider(CommonDataProviderXl.java:16)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- commonDataProvider -->
        <test-method signature="commonDataProvider(java.util.Hashtable)[pri:0, instance:endToEndAutomation.CommonDataProviderXl@1b2abca6]" started-at="2022-01-31T23:25:32 PKT" name="commonDataProvider" data-provider="dp" finished-at="2022-01-31T23:25:32 PKT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{class1=test5, name=test4, code=test6, runmode=Y}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- commonDataProvider -->
        <test-method signature="commonDataProvider(java.util.Hashtable)[pri:0, instance:endToEndAutomation.CommonDataProviderXl@1b2abca6]" started-at="2022-01-31T23:25:32 PKT" name="commonDataProvider" data-provider="dp" finished-at="2022-01-31T23:25:32 PKT" duration-ms="0" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{class1=test5, name=test4, code=test6, runmode=N}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[with N no]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: with N no
at endToEndAutomation.CommonDataProviderXl.commonDataProvider(CommonDataProviderXl.java:16)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- commonDataProvider -->
      </class> <!-- endToEndAutomation.CommonDataProviderXl -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
